js/script.js

```diff
@@ -67,6 +67,17 @@ const createTaskCard = task => {
     return taskCard;
 };
 
+const isMatchingId = (task, taskId) => {
+    return task.id === taskId;
+};
+
+const deleteTask = taskId => {
+    const index = tasks.findIndex(task => isMatchingId(task, taskId));
```

Comment:
Do we really need an extra function `isMatchingId`?

can't we only write like this?
```suggestion
    const index = tasks.findIndex(task => task.id === taskId);
```